demos/fleet_provisioning/fleet_provisioning_with_csr/CMakeLists.txt

```diff
@@ -10,6 +10,19 @@ include( ${CMAKE_SOURCE_DIR}/libraries/standard/backoffAlgorithm/backoffAlgorith
 include(
     ${CMAKE_SOURCE_DIR}/libraries/aws/fleet-provisioning-for-aws-iot-embedded-sdk/fleetprovisioningFilePaths.cmake )
 
+# Set path to corePKCS11 and it's third party libraries.
+set(COREPKCS11_LOCATION "${CMAKE_SOURCE_DIR}/libraries/standard/corePKCS11")
+set(CORE_PKCS11_3RDPARTY_LOCATION "${COREPKCS11_LOCATION}/source/dependency/3rdparty")
+
+# Include PKCS #11 library's source and header path variables.
+include( ${COREPKCS11_LOCATION}/pkcsFilePaths.cmake )
+
+list(APPEND PKCS_SOURCES
```

Comment:
Seems like these extra sources can be added under `PKCS_POSIX_PAL_SOURCES` and `PKCS_WINSIM_PAL_SOURCES` within corePKCS11

Reply:
This can be done later.